docs/components.md

```diff
@@ -21,28 +21,29 @@ KubeVirt consists of a set of services:
 
 ## Example flow: Create and Delete a VM
 
-<!-- FIXME: wrong in a lot of ways and therefore misleading -->
-
-    User       Virt API      VM TPR    Virt Controller    k8s      VM Pod
-    createVM  --> |                           |            |
-                  o create --> +              |            |
-                  |            | <~ create ~> ?            |
-                  |            |              |            |
-                  |            |              o create --> o ------> +
-                  |            |              |            |         |
-                  |            | <---------------------- get VM Spec o
-                  |            o ----------------------------------> |
-                  |            |              |            |         o defineVM()
-                  |            |              |            |         o watchVM()
-                  |            |              |            |         |
-    deleteVM -->  o delete --> *              |            |         |
-                  |              <~ delete ~> ?            |
-                  |                           |            |         |
-                  |                           o delete --> o ------> *
-                  |                           |            |
-                  :                           :            :
-
-    Legend: ?: Event notification
+    Client        Virt API      VM TPR  Virt Controller    k8s      VM Pod
+    ------------- ------------- ------- ------------------ -------- ------
+
+                  listen <------------- WATCH /vms
+                                           |
+    POST /vms ----> validate               |
+                    create ---> VM ---> observe
+                      |                    v
+                      |                 POST /pods ----> validate
+                      |                    |             create --> Pod
```

Comment:
From here the "Virt Controller" sees that the pod was started and assigns the VM to a node. Now that the VM is assigned to a node, virt-launcher sees the VM and can handle it. So virt-handler only has to watch for VMs which are assigned to it